### PR TITLE
creating calendar component

### PR DESCRIPTION
**Pull request by sharleenawinja**
# Adding calendar to add patients component

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/calendar/Calendar.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+interface calendarProps {
+  selectedMonth: string;
+  // eslint-disable-next-line no-unused-vars
+  handleMonthChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```
Comment (kipchubett) [inline src/components/calendar/Calendar.tsx]: Are you using `handleMonthChange`?

File: src/components/calendar/Calendar.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        className="px-3 py-2 border border-gray-400 rounded-lg"
+        min="2020-01"
+        value={selectedMonth}
+        onChange={handleMonthChange}
+      />
+    </div>
+  );
+};
+
+export default Calendar;
```
Comment (kipchubett) [inline src/components/calendar/Calendar.tsx]: Is this support in all browsers? try Firefox

File: src/components/calendar/Calendar.tsx (hunk 1 above)
Comment (sharleenawinja) [reply src/components/calendar/Calendar.tsx]: I'm using it to set the selected month as the reporting month

File: src/components/calendar/Calendar.tsx (hunk 2 above)
Comment (sharleenawinja) [reply src/components/calendar/Calendar.tsx]: I've tried and it seems like it is not supported by firefox

Comment (kipchubett) [reply src/components/calendar/Calendar.tsx]: What's your recommendation on this?

Comment (sharleenawinja) [reply src/components/calendar/Calendar.tsx]: I'll keep on looking at other options

Comment (kipchubett) [reply src/components/calendar/Calendar.tsx]: https://www.npmjs.com/package/react-calendar

**Review by kipchubett — APPROVED** [review]
LGTM!

**Comment by kipchubett** [comment]
Can you look into this build error
```shell
src/app/rde-sync/Moh731Sync.component.tsx(15,34): error TS2307: Cannot find module 'date-fns' or its corresponding type declarations.
```

**Pull request merged**